e2e-tests/tfm/cypress.config.js

```diff
@@ -23,23 +23,38 @@ module.exports = defineConfig({
   referenceDataApiPort: '5002',
   apiKey: TFM_API_KEY,
   dbName: 'dtfs-submissions',
+  redisHost: 'localhost',
+  redisPort: '6379',
+  redisKey: '',
   dbConnectionString: 'mongodb://root:r00t@localhost:27017/?authMechanism=DEFAULT&directConnection=true',
   responseTimeout: 100000,
   pageLoadTimeout: 120000,
   redirectionLimit: 100,
   numTestsKeptInMemory: 1,
-  viewportWidth: 3840,
+  viewportWidth: 1450, // TFM website max width is 1440px
```

Comment:
We should change this to be 1440p (changing height) in which case

Reply:
ah I see this now -- Effectively there's no use having this wider. I've changed this from `1450` with an odd viewport height to just being 1080p. I don't have an issue reverting this to 4k, but I feel most devs have 1080p